src/content/protocols/aave-v3-etherfi.md

```diff
@@ -0,0 +1,518 @@
+---
+protocol: "Aave v3 EtherFi"
+website: "https://aave.com"
+x: "https://x.com/aave"
+github: ["https://github.com/aave-dao/aave-v3-origin"]
+defillama_slug: ["aave-v3"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "H", "H", "H", "L"]
+reasons: []
+author: ["sagaciousyves"]
+submission_date: "2025-05-16"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+On Ethereum Mainnet, 3 different instances exist _Core_, _Prime_ and _EtherFi_ which all are managed by the Aave DAO. **This report covers the Aave V3 _EtherFi_ instance.** To check out our main report on Aave V3 see the [_Core Instance_ report](/protocols/aave). This report focuses on the key differences between the _EtherFi_ and the _Core_ instance.
+
+Aave v3 is a lending protocol that allows users to lend and borrow different ERC20 assets. Users are able to create positions that consist of debt in different loan assets which is secured by different collateral assets. The lending market allows anyone to liquidate insolvent positions, based on an external price feed and specific collateral factors representing an asset's specific risk profile. Furthermore, instead of borrowing supplied assets, Aave V3 also issues its own stablecoin, `GHO`. Users can borrow and lend `GHO` like any other asset in the system.
+
+# Ratings
+
+## Chain
+
+The report is concerned with the Aave V3 _EtherFi_ instance deployed on Ethereum mainnet. Ethereum mainnet achieves a _Low_ centralization risk score.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Aave v3 protocol can be organized in a number of logical modules: _Core Lend & Borrow_, _Reserve Parameters_, _Treasury, Aave Ecosystem Reserves & Rewards_, _GHO Stablecoin_ and _Aave Governance_. Each module exposes various degrees of control as explained in more detail below. Overall, these control vectors could result in the loss of user funds, loss of unclaimed yield or otherwise materially affect the expected performance of the protocol.
```

Comment:
```suggestion
The Aave v3 protocol can be organized in a number of logical modules: _Core Lend & Borrow_, _Reserve Parameters_, _Treasury, Aave Ecosystem Reserves & Rewards_, _GHO Stablecoin_, and _Aave Governance_. Each module exposes various degrees of control as explained in more detail below. Overall, these control vectors could result in the loss of user funds, loss of unclaimed yield, or otherwise materially affect the expected performance of the protocol.
```

"Explained in more details below" => Where? I don't think this is part of the protocol analysis here